chain/beacon/drand/drand.go

```diff
@@ -218,6 +223,10 @@ func (db *DrandBeacon) maxBeaconRoundV2(latestTs uint64) uint64 {
 	// we take the time from genesis divided by the periods in seconds, that
 	// gives us the number of periods since genesis.  We also add +1 because
 	// round 1 starts at genesis time.
+	// TODO: Confirm that we are still fetching the right round for Filecoin
```

Comment:
Gonna experimentally confirm this, but special review here is requested.

Reply:
This _should_ give us the drand beacon generated when `latestTs` was produced, so yes?

Reply:
Are we sure we want "at least 30 seconds"? We didn't actually have the option of forcing propagation delays before... (probably too late to bikeshed this now).

Reply:
I asked Protocol Design gang about this, and they basically said "leave at 30 for now, change later maybe" (which is kinda how I feel about it too).

Reply:
Confirmed this manually, dropping the TODO.